### PR TITLE
Update rooms.json

### PR DESCRIPTION
**Pull request by LadyAbigor**

### CODE REVIEW

File: app/config/tstvhq/rooms.json [hunk 1]
```diff
@@ -261,7 +261,10 @@
                 "src": "https://video.nest.com/embedded/live/UYYI6tFUy2?autoplay=1"
             },
             "directions": {
-                "south": "theater"
+                "south": "
+                
+
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json]: this is definitely wrong, was this an accident?

**Review by hndrewaall — CHANGES_REQUESTED** [review]

**Review by LadyAbigor — COMMENTED** [review]
Corrected 264

File: app/config/tstvhq/rooms.json [hunk 2]
```diff
@@ -261,7 +261,8 @@
                 "src": "https://video.nest.com/embedded/live/UYYI6tFUy2?autoplay=1"
             },
             "directions": {
-                "south": "theater"
+                "south": "
+                "
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json]: still wrong. what are you trying to do here?

**Review by hndrewaall — CHANGES_REQUESTED** [review]

File: app/config/tstvhq/rooms.json (hunk 1 above)
Comment (LadyAbigor) [reply app/config/tstvhq/rooms.json]: Removed the extra "

File: app/config/tstvhq/rooms.json [hunk 3]
```diff
@@ -261,7 +261,7 @@
                 "src": "https://video.nest.com/embedded/live/UYYI6tFUy2?autoplay=1"
             },
             "directions": {
-                "south": "theater"
+            "south": "theater",
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json]: almost 😅 
```suggestion
            "south": "theater"
```

**Review by hndrewaall — APPROVED** [review]

**Pull request merged**